draft-ietf-oauth-cross-device-security.md

```diff
@@ -124,12 +124,12 @@ carry with them.
 
 There are three cross-device flow patterns for transferring the authorization request between the Initiating Device to the Authorization Device.
 
-- User transferred: In the first variant, the user initiates the authorization process with the authorization server by copying information from the initiating device to the authorization device, before authorizing an action. For example the user may read a code displayed on the initiating device and enter it on the authorization device, or they may scan a QR code displayed in the initiating device with the authorization device.
+- User Transferred Session Data Pattern: In the first variant, the user initiates the authorization process with the authorization server by copying information from the initiating device to the authorization device, before authorizing an action. For example the user may read a code displayed on the initiating device and enter it on the authorization device, or they may scan a QR code displayed in the Initiating Device with the Authorization Device.
```

Comment:
I wonder if we should hyphenize ("User-Transferred Session Data Pattern"). WDYT?

Reply:
It think that makes sense. What is the best way for me to do that (accept the current PR, and then do a search and replace)?

Reply:
OK, I think I made the changes to user-transferred. I wonder if we need the same for client transferred. Thoughts?